### PR TITLE
EVG-14829 inactive commits tooltip component 

### PR DESCRIPTION
**Pull request by khelif96**
[EVG-14829](https://jira.mongodb.org/browse/EVG-14829)

### Description 
Add inactive commits tooltip component. This depends on a `githash` field that we need to add to the version resolver. 

### Screenshots
image

### CODE REVIEW

File: src/pages/commits/InactiveCommits/index.tsx [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+      }`}</HiddenCommitsWrapper>
+    );
+    returnedCommits = [
+      ...returnedCommits,
+      ...rolledUpVersions.slice(-3).map((v) => (
+        <CommitText key={v.githash} data-cy="commit-text">
+          {getCommitCopy(v)}
+        </CommitText>
+      )),
+    ];
```
Comment (SupaJoon) [inline src/pages/commits/InactiveCommits/index.tsx:51]:
```suggestion
      returnedCommits = [
      ...rolledUpVersions.slice(0, 2).map((v) => (
        <CommitText key={v.githash} data-cy="commit-text">
          {getCommitCopy(v)}
        </CommitText>
      )),
      <HiddenCommitsWrapper
        key="hidden_commits"
        data-cy="hidden-commits"
      >{`${hiddenCommitCount} more commit${hiddenCommitCount !== 1 ? "s" : ""
        }`}</HiddenCommitsWrapper>,
      ...rolledUpVersions.slice(-3).map((v) => (
        <CommitText key={v.githash} data-cy="commit-text">
          {getCommitCopy(v)}
        </CommitText>
      )),
    ];
```

**Review by SupaJoon — APPROVED** [review]

File: src/pages/commits/InactiveCommits/InactiveCommits.test.tsx [hunk 2]
```diff
@@ -0,0 +1,122 @@
+import userEvent from "@testing-library/user-event";
+import { render, act } from "test_utils/test-utils";
+
+import { InactiveCommits } from ".";
+
+const RenderInactiveCommits = (versions) => (
```
Comment (SupaJoon) [inline src/pages/commits/InactiveCommits/InactiveCommits.test.tsx:6]: this is a React component that accepts the same props as InactiveCommits. I think we can replace RenderInactiveCommits usage with <InactiveCommits />

File: src/pages/commits/InactiveCommits/index.tsx [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+    returnedCommits = rolledUpVersions.map((v) => (
+      <CommitText key={v.githash} data-cy="commit-text">
+        {getCommitCopy(v)}
+      </CommitText>
+    ));
+  }
+
+  return (
+    <Tooltip
+      align="top"
```
Comment (sophstad) [inline src/pages/commits/InactiveCommits/index.tsx:62]: Based on the Figma and playing around with the Tooltip component [on mongodb.design](https://www.mongodb.design/component/tooltip/example/), I think align should be set to `bottom`

**Review by sophstad — APPROVED** [review]
Nice! Just one note~

**Pull request merged**